airbyte-webapp/src/views/layout/SideBar/components/BuilderIcon.tsx

```diff
@@ -0,0 +1,10 @@
+const BuilderIcon = ({ color = "currentColor" }: { color?: string }): JSX.Element => (
```

Comment:
Depending if this or https://github.com/airbytehq/airbyte/pull/22433 gets merged first, we should make sure to adjust the other open PR to use the `Icon` components or add this icon to the icon component respectively. cc @krishnaglick

Reply:
Sounds good